package.json

```diff
@@ -15,9 +15,9 @@
     "@google-cloud/vision": "^2.3.2",
     "@mux/blurhash": "^0.1.2",
     "@mux/mux-node": "^7.0.0",
-    "@mux/mux-player-react": "^1.7.1",
+    "@mux/mux-player-react": "^1.8.0",
     "@mux/mux-uploader-react": "^1.0.0-beta.5",
```

Comment:
did we want to upgrade this package too? there's a beta.6 now.

Reply:
I'll leave that for next time